### DISCUSSION TITLE
License blocked on Firefox

### DISCUSSION THREAD

**Discussion by Chaps145** [aclap-dev/video-downloadhelper#1556] [Help and Q&A]
### Your question

Hi !
After reinstalling Firefox and vdh it appears that my lifetime license is blocked. 

I am no longer sure of the email address with which I subscribed to this license but I have not received any emails in my various inboxes anyway. 

Could you tell me how to unlock my license on Firefox?

Thanks !

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: fr
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"blocked":true,"key":"4cd5514799c24691"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 1
}

**Comment by jcfrog** [comment]
Hi,
You should receive an email to unlock it. Maybe in your spam folder.
If not, send us an email at licenses@downloadhelper.net, we'll try to resolve the problem.

**Reply by Chaps145** [reply]
Hello, Thank you for your reply. I sent an email because I did not receive anything in my inbox or spam.
